package.json

```diff
@@ -23,15 +23,18 @@
   "contributors": [
     "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)"
   ],
+  "types": "types/index.d.ts",
   "files": [
+    "types/index.d.ts",
     "index.js"
   ],
   "dependencies": {
-    "hast-util-sanitize": "^2.0.0"
+    "hast-util-sanitize": "^3.0.0"
```

Comment:
```
  types/index.d.ts:3:22

  ✖  3:22  unified should be listed in the project's dependencies. Run npm i -S unified to add it  import/no-extraneous-dependencies
```

https://travis-ci.org/github/rehypejs/rehype-sanitize/jobs/713293480#L233-L234

Reply:
OK. Shall we add it to `dependencies`? It seems more like a `peerDependencies`

Reply:
unified has primarily used `dependencies` for types directly used, and `devDependencies` for types only used for testing.
`peerDependencies` can be problematic due to inconsistent handling across dependency managers, and even within versions of the same dependency manager.

In this case I'd lean toward `dependencies`.